src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx

```diff
@@ -25,6 +25,10 @@ redirects:
   - /docs/platform/platform-terminology
   - /docs/more-integrations/open-source-telemetry-integrations
   - /docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic-integrations
+  - /docs/more-integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations
```

Comment:
@zuluecho9, I think this document is on the short-list for removal, but we can still grab these redirects if this doc is  ever deleted.